activestorage/README.md

```diff
@@ -16,6 +16,8 @@ A key difference to how Active Storage works compared to other attachment soluti
 
 Run `rails active_storage:install` to copy over active_storage migrations.
 
+NOTE: If the rake task cannot be found, verify that `require "active_storage/engine"` is present in `config/application.rb`.
```

Comment:
Hi @donniepropst! Thanks for this note! Just one comment, I would rather remove rake from it and only refer to task since all this tasks/commands were unified as `rails` subcommands. Referring to rake can could cause confusion (note that the message you get doesn't).

Something like: `If the task cannot be found` would be great.

Thanks!

Reply:
Fixed 😄